corehq/motech/requests.py

```diff
@@ -26,9 +31,23 @@
 )
 from corehq.util.view_utils import absolute_reverse
 
+MAX_TRIES = 3
+DEFAULT_EXPONENTIAL_PERIOD = 'minutes'
+DEFAULT_BACKOFF_RETRY_EXPONENT_BASE = 2
+SECONDS_IN_ONE_SECOND = 1
+SECONDS_IN_ONE_MINUTE = 60
+SECONDS_IN_ONE_HOUR = 3600
+SECONDS_IN_ONE_DAY = 86400
 
-def log_request(self, func, logger):
+SECONDS_IN_PERIOD = {
+    'seconds': SECONDS_IN_ONE_SECOND,
+    'minutes': SECONDS_IN_ONE_MINUTE,
+    'hours': SECONDS_IN_ONE_HOUR,
+    'days': SECONDS_IN_ONE_DAY
+}
```

Comment:
These are the defaults necessary for generic request backoff function (to be used as decorator)